presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -208,7 +208,7 @@ public void testArrayToArrayConcat()
         assertInvalidFunction("ARRAY [ARRAY[1]] || ARRAY[ARRAY[true], ARRAY[false]]", FUNCTION_NOT_FOUND);
 
         // This query is ambiguous. The result can be [[1], NULL] or [[1], [NULL]] depending on interpretation
-        assertFunction("ARRAY [ARRAY [1]] || ARRAY [NULL]", new ArrayType(new ArrayType(INTEGER)), asList(ImmutableList.of(1), null));
+        // assertFunction("ARRAY [ARRAY [1]] || ARRAY [NULL]", new ArrayType(new ArrayType(INTEGER)), asList(ImmutableList.of(1), null));
```

Comment:
For this expressing 3 different methods could be invoked correctly:
`concat<E>(E,array(E)):array(E)`, `concat<E>(array(E),E):array(E)`, `concat<E>(array(E),array(E)):array(E)`. Why do we consider that the latest method is the most specific and must be chosen?

In java